developers/bindings/index.md

```diff
@@ -631,7 +631,11 @@ This is to support the old DSL rules engine and make the action available there.
 
 Each parameter of an action member method must be annotated with `@ActionInput`.
 
-If you return values, you do so by returning a `Map<String,Object>` and annotate the method itself with as many `@ActionOutput`s as you will return map entries.
+If you return values, you do so by returning a `Map<String,Object>` and annotate the method itself with `@ActionOutputs` containing as many `@ActionOutput`s as you will return map entries, e.g.
```

Comment:
Perhaps it should be clarified how to handle when this is dynamic? For example:

https://github.com/openhab/openhab-addons/blob/3cd5879209f40f0af8991d11b21ab748d31978f2/bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java#L118

https://github.com/openhab/openhab-addons/blob/3cd5879209f40f0af8991d11b21ab748d31978f2/bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java#L129-L138

I would assume that all possible return values should be annotated, in this simple case `CheapestStart` and `MostExpensiveStart`?